web/scores.go

```diff
@@ -98,18 +159,27 @@ func (web *Web) setScoresHandler(w http.ResponseWriter, r *http.Request) {
 		handleWebErr(w, err)
 		return
 	}
+
+	var scoresMap map[string]interface{}
+
+	// ick - is there a way to parse into the struct - and tell which fields are explicitly filled?
```

Comment:
> is there a way to parse into the struct - and tell which fields are explicitly filled?

I think you can use pointer fields in the struct. if the field was not present in the JSON, the field will be nil